recipe/build.sh

```diff
@@ -56,9 +56,10 @@ function mkcp {
 # First make a few of the libraries to check that they work!
 make libxmlparser.a
 # Try and build FoX to catch any debugs
-make FoX/.config || cat FoX/config.log
+# make FoX/.config || cat FoX/config.log
 
 mkcp siesta
+make clean-transiesta
 mkcp transiesta
```

Comment:
In 4.1 you don't need this. I will fix this.

Reply:
Then I get the following error message: 
```
RUNNING: make transiesta
** You were building siesta in this directory.
** Please type 'make clean-transiesta' first 
make: *** [check-transiesta] Error 1
Traceback (most recent call last):
  File "/opt/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 474, in main
    execute(sys.argv[1:])
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 465, in execute
    verify=args.verify, variants=args.variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2864, in build_tree
    notest=notest,
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1995, in build
    cwd=src_dir, stats=build_stats)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/utils.py", line 404, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/utils.py", line 384, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'err
```

Reply:
Sorry, I wasn't clear. Transiesta command is not necessary. :)
So `make clean-transiesta`, `make transiesta` is not necessary. They can be emulated using a simple shell script
```bash
#!/bin/sh
siesta --electrode $@
```